Assets/Scripts/Data/Pickup Pup/DogDatabase.cs

```diff
@@ -21,7 +24,7 @@ static Sprite defaultSprite
 			} 
 			else 
 			{
-				// Memoization for effeciency :
+                // Memorization for efficiency
```

Comment:
Admittedly I can't spell efficiency, but memoization is a real thing: https://en.wikipedia.org/wiki/Memoization